### PR TITLE
Fix/authorization pending

### PR DESCRIPTION
**Pull request by justAnIdentity**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -8,17 +10,15 @@ use rocket::{
 use rocket_dyn_templates::Template;
 use ssi::jwk::{Params, JWK};
 
-use oidc4vci_issuer::*;
-
 #[rocket::post("/token", data = "<query>")]
 fn post_token_default_op_state(
-    query: rocket::form::Form<token::TokenQueryParams>,
+    query: rocket::form::Form<TokenRequest>,
```
Comment (cobward) [inline src/main.rs]: If the intention is for this to be a generic on-spec oid4vci issuer, I wonder whether we should leave this as TokenQueryParams and just define TokenRequest in the microservice?

Comment (sbihel) [reply src/main.rs]: Agreed. And in the future oid4ci-rs will define a struct with additional properties generics but for now just copy paste TokenRequest into the microservice

Comment (justAnIdentity) [reply src/main.rs]: Ok, will do

File: src/token.rs [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
 use std::collections::HashMap;
 
 use crate::{error::Error, types::Metadata};
 
+#[derive(Debug, FromForm, Deserialize, Serialize)]
+pub struct TokenRequest {
+    #[serde(flatten)]
+    pub inner: TokenQueryParams,
+    pub authorization_pending: bool,
+}
```
Comment (cobward) [inline src/token.rs]: Can this be deleted?

**Review by cobward — APPROVED** [review]
Minor comment but otherwise LGTM

**Pull request merged**